index.js

```diff
@@ -5,6 +5,7 @@ const path = require("path");
 const https = require("https");
 const tar = require('tar');
 const os = require('os');
+const packageJson = require('./package.json');
```

Comment:
Has this been checked locally using npm link to check this works

Reply:
why npm link? I've run `node index.js -v` and it works returning this
![obraz](https://github.com/danielglejzner/ng16-dep-audit/assets/81327266/061c5d7e-d432-4f88-904e-f50c44b5645f)

Reply:
Np link crates it as a local package to run rather than locally

Reply:
I've link the repo and run the command in another repo and it looks good

![obraz](https://github.com/danielglejzner/ng16-dep-audit/assets/81327266/2c5238b5-3053-4427-b2f3-8c05493b1e01)